code/modules/events/infestation.dm

```diff
@@ -26,11 +26,22 @@
 		INFESTATION_SPACE_BATS = 1,
 		INFESTATION_SPIDERLINGS = 1
 	)
-/datum/event/infestation/major
-	possible_mobs = list(
-		INFESTATION_HIVEBOTS = 1,
-		INFESTATION_SLIMES = 1
-	)
+
+/datum/event/infestation/major/setup()
+	var/player_count = 0
+	for(var/mob/living/carbon/human/H in living_mob_list)
+		if(H.stat == CONSCIOUS && H.client)
+			player_count++
```

Comment:
why check for consciousness instead of just `H.stat != DEAD`?

Reply:
From my limited understanding, those that are dead are already removed from living_mob_list, and checking consciousness would discard those that are out of action in another way.